### PR TITLE
SC 18 overview of devices

### PR DESCRIPTION
**Pull request by adamjhr**
Do not merge

### CODE REVIEW

File: docker/grafana/alerting/1/__default__.tmpl
Comment (DanyalYor) [inline docker/grafana/alerting/1/__default__.tmpl:1]: Is this an automatically generated file? Can we have it excluded from the repo?

File: docker/docker-compose.yaml
Comment (DanyalYor) [inline docker/docker-compose.yaml:1]: Why are we having a dedicated Dockerfile for grafana that just builds an image already available on the hub?

Comment (DanyalYor) [reply docker/docker-compose.yaml:1]: The same can be said about prometheus, but here we have a volume binding. Can this also not just be part of the docker compose specification?

**Review by DanyalYor — CHANGES_REQUESTED** [review]
Please check the comments :)
Other than that, lgtm!

File: docker/grafana/alerting/1/__default__.tmpl
Comment (jfho) [reply docker/grafana/alerting/1/__default__.tmpl:1]: Seems so, I'll add it to the .gitignore

File: docker/docker-compose.yaml
Comment (jfho) [reply docker/docker-compose.yaml:1]: Yes, they should be able to run without their own Dockerfiles, seems like we can have most settings in the docker compose instead.

**Review by DanyalYor — APPROVED** [review]

**Pull request merged**